mkt/reviewers/helpers.py

```diff
@@ -122,6 +123,13 @@ def queue_tabnav(context):
              .format(counts['moderated'])),
         )
 
+    if acl.action_allowed(request, 'Apps', 'AbuseReports'):
```

Comment:
Typically we've named these as None:Verb, like `Apps:ReportAbuse`, or something? Check out the others and see if there's a pattern we can follow.